react/src/utilities/helpers/isSprkInput/isSprkInput.js

```diff
@@ -0,0 +1,9 @@
+const isSprkInput = (elem) => {
+  return (
```

Comment:
This function will throw an error if elem doesnt have the type property so we need an early return added in this function to check for that.